components/engine_traits/src/engine.rs

```diff
@@ -153,32 +153,79 @@ impl Drop for TabletErrorCollector {
     }
 }
 
+#[derive(Default)]
+pub struct OpenOptions {
```

Comment:
You may need a `skip_cache` as open_raw should not update cache.

Reply:
Or perhaps just keep `open` and `open_raw` as `open_raw` needs a `Path` instead of `region_id` and `suffix`.

Reply:
Okay.